api-service/src/main/java/com/bitmark/apiservice/middleware/Converter.java

```diff
@@ -84,7 +85,7 @@ public void onSuccess(Response response) {
                 try {
                     String raw = response.body().string();
                     Map<String, Object> json = jsonToMap(raw);
-                    callback.onSuccess(json.get("txid").toString());
+                    callback.onSuccess(Optional.ofNullable(json.get("txid")).orElse(json.get("txId")).toString());
```

Comment:
I concern that `Optional` is not supported in lower android version. Could you write it in simple way?